### PR TITLE
refactor UpdateMessage

### PR DESCRIPTION
**Pull request by akash1810**
Make `id` required as it should always be present. Also remove `withImageId` as a result too.

Easier to review here - https://github.com/guardian/grid/pull/2491/files?w=1

### CODE REVIEW

File: thrall/app/lib/kinesis/ThrallEventConsumer.scala [hunk 1]
```diff
@@ -48,26 +49,30 @@ class ThrallEventConsumer(es: ElasticSearchVersion,
...
-              Logger.info("Completed processing of update message: " + messageLogMessage)
-            }.recover {
-              case e: Throwable =>
-                Logger.error("Failed to process update message; message will be ignored: " + ("Got update message: " + messageLogMessage), e)
-            }
+          val maybeUpdateMessage = Json.parse(message).validate[UpdateMessage] <| logParseErrors |> (_.asOpt)
 
-            Await.ready(eventuallyAppliedUpdate, ThirtySeconds)
-          }
+          maybeUpdateMessage match {
```
Comment (akash1810) [inline thrall/app/lib/kinesis/ThrallEventConsumer.scala]: fail fast - validate incoming message against `UpdateMessage`

**Comment by tonytw1** [comment]
Seem to recall the 'heartbeat' message been the sticking point. Is it compatible with the mandatory id or can it been deleted?

**Comment by akash1810** [comment]
@tonytw1 great point and you're absolutely right! I think we can change the shape of the heartbeat message to include an id to address this.

**Comment by akash1810** [comment]
On second thoughts, we can be better still and deserialise into the original case classes rather than a generic `UpdateMessage`. Closing this PR and will raise another to that effect.

**Pull request closed without merging**